.travis.yml

```diff
@@ -17,7 +17,7 @@ notifications:
     - thomas_d_j@yahoo.com.au
 
 script:
-    - scons VERBOSE=1
+    - scons VERBOSE=1 DEBUG=1
```

Comment:
As discussed earlier, I'd love to have TravisCI still run with an optimized build (which ``DEBUG=1`` sets to ``-O0``).
Suggestion: Make DEBUG=1 still honor the ``O=2`` switch when it's set explicitly.